src/types/shared.ts

```diff
@@ -1,3 +1,15 @@
+type Permit = {
```

Comment:
here's the type I have for lend which is slightly different, we should try to combine into a shared one now.

type AccountPermit = {
  params: {
    data?: string,
    contract?: string,
    key: string,
  },
  chain_id : string,
  signature : StdSignature,
}

Reply:
import { StdSignature } from 'secretjs';